OracleWebLogic/samples/12212-domain/container-scripts/createServer.sh

```diff
@@ -2,11 +2,24 @@
 #
 # Copyright (c) 2014-2015 Oracle and/or its affiliates. All rights reserved.
 #
+export DOMAIN_HOME=/u01/oracle/user_projects/domains/$DOMAIN_NAME
+echo "Domain Home: " $DOMAIN_HOME
+
+if [ -z $ADMIN_PASSWORD ]; then
+   echo "      ----> NO 'weblogic' admin password set"
+   echo ""
+else
+   s=${ADMIN_PASSWORD}
+   echo "      ----> 'weblogic' admin password: $s"
+fi
+sed -i -e "s|ADMIN_PASSWORD|$s|g" /u01/oracle/commonfuncs.py
```

Comment:
Avi the problem here is that the AdminServer container as well as Managed Server containers all need to use the same exact Admin Password.  I auto generate the password but I need to allow the users override that auto generated password to be anything they want to set.